pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computefirewallpolicyrule/computefirewallpolicyrule-minimal/_http.log

```diff
@@ -208,14 +209,14 @@ X-Xss-Protection: 0
 
 ---
 
-POST https://www.googleapis.com/compute/v1/locations/global/firewallPolicies/${firewallPolicyID}/addRule?alt=json
+POST https://compute.googleapis.com/compute/v1/locations/global/firewallPolicies/${firewallPolicyID}/addRule
 Content-Type: application/json
-User-Agent: kcc/controller-manager DeclarativeClientLib/0.0.1
+User-Agent: kcc/controller-manager
+x-goog-request-params: firewall_policy=${firewallPolicyId}
 
 {
   "action": "deny",
   "direction": "INGRESS",
-  "firewallPolicy": "locations/global/firewallPolicies/${firewallPolicyId}",
```

Comment:
Direct controller does not have `firewallPolicy` in request body, while DCL controller does. 
In the API [doc](https://cloud.google.com/compute/docs/reference/rest/v1/firewallPolicies/addRule), `firewallPolicy` is path parameter, not a field in the request body. I think this diff is acceptable.